go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.26.1
 	github.com/robfig/cron v1.2.0
 	github.com/spectrocloud/hapi v1.14.1-0.20230428134500-8a894264c61c
-	github.com/spectrocloud/palette-sdk-go v0.0.0-20230502044012-10512e17ec52
```

Comment:
@SivaanandM could you please resolve conflicts?

Reply:
resolved the conflict